pantheon-bundle/src/main/java/com/redhat/pantheon/validation/validators/ValidationMessageHelper.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.pantheon.validation.validators;
+
+import org.osgi.framework.BundleContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Properties;
+
+public class ValidationMessageHelper {
```

Comment:
This file is interesting - what is the intention here?

Is it so that we can change the wording of messaging without needing to rebuild and redeploy?

If so, isn't the properties file also part of the bundle, which would mean we'd need to redeploy anyways in order to update it?